aks/k8sresources.tf

```diff
@@ -1,39 +1,11 @@
 provider "kubernetes" {
-  load_config_file = false
-  version                = "1.10.0"
-  host                   = "${azurerm_kubernetes_cluster.k8s.kube_config.0.host}"
-  client_certificate     = "${base64decode(azurerm_kubernetes_cluster.k8s.kube_config.0.client_certificate)}"
-  client_key             = "${base64decode(azurerm_kubernetes_cluster.k8s.kube_config.0.client_key)}"
-  cluster_ca_certificate = "${base64decode(azurerm_kubernetes_cluster.k8s.kube_config.0.cluster_ca_certificate)}"
-}
-
-resource "kubernetes_service_account" "tiller" {
-  metadata {
-    name = "tiller"
-    namespace = "kube-system"
-  }
-
-  automount_service_account_token = true
 
- // depends_on = ["null_resource.post_processor"]
-  depends_on = ["azurerm_kubernetes_cluster.k8s"]
-}
-
-resource "kubernetes_cluster_role_binding" "tiller" {
-    metadata {
-        name = "tiller"
-    }
-    role_ref {
-        api_group = "rbac.authorization.k8s.io"
-        kind = "ClusterRole"
-        name = "cluster-admin"
-    }
-    subject {
-        kind = "ServiceAccount"
-        name = "tiller"
-        namespace = "kube-system"
-    }
-    depends_on = ["kubernetes_service_account.tiller"]
+  version = "~> 1.10.0"
```

Comment:
is this the k8s version? its not clear what version will be used.

Reply:
No, this is the Terraform kubernetes provider version.